WordPress/src/androidTest/java/org/wordpress/android/ui/screenshots/WPScreenshotTest.java

```diff
@@ -62,7 +62,9 @@ public void wPScreenshotTest() {
 
             wpLogin(); // TODO commented for quicker debugging
 
-            editBlogPost();
+            // Disabled because it keeps failing with the `AppNotIdleException` caused by an infinite auto-save loop
```

Comment:
I am seeing this same issue if I uncomment the `editBlogPost()` test. :(

Reply:
> I am seeing this same issue if I uncomment the editBlogPost() test. :(

It's an interesting one that will need further investigation. Could be either something off in the app code (which I'd assume is more unlikely) or something off coming from the request mocks / some specifics in the request mocks that's making the app refresh the list again and again.

This will be for a bit later, presumably after the Jetpack screenshots UI tests PR is open.
Main reason for planning it this way is that it does seem to be a bigger issue that's well defined, we can dedicate all focus only on that problem while at it 👍 .